readme/apps/mfa.md

```diff
@@ -0,0 +1,59 @@
+# Multi-factor authentication
+
+You can now enable Multi-Factor Authentication (MFA) to your Joplin Account.
+MFA enhances your account security by requiring more than just a password to log in.
+Joplin Cloud uses a method that generates a new code every 30s, this code needs 
+to be entered on each new log in after MFA is enabled.
+
+If you can't generate your time-based code because you have lost access to your 
+smartphone, for example, you will need to use one of the recovery codes to log in.
+
+## Recovery Codes
+
+When you enable MFA, Joplin Cloud will show you a list of codes,that you should save in 
+somewhere safe. Each code can only be used once, but you can generate new codes,
+if necessary, from your profile page.
+
+### How to enable MFA
```

Comment:
All the `###` should be `##` (since they are not all a sub-section of `Recovery codes`)